docs/software/android/usage.mdx

```diff
@@ -106,14 +106,24 @@ You can test changing channels with the QR code shown below.
 You can change the following settings by tapping on the name of an existing channel:
 1. MQTT Uplink
 2. MQTT Downlink
-3. Position precision value
+3. Position enabled 
+4. Precise location (if position is enabled)
+5. Position precision (if precise location is disabled)
 
 #### Position Precision
 The precision levels currently used in the app correspond to the following values:
-- HIGH_PRECISION (32), full precision
-- MED_PRECISION (16), around ±350 meters
-- LOW_PRECISION (11), around ±11 kilometers
-- DISABLED (0), location data is never sent over the given channel
+| Precision bits | Approx. Uncertainty (±) |
```

Comment:
This table is awesome, what if we put it under the [channel config doc](https://meshtastic.org/docs/configuration/radio/channels/#position-precision)? Then so we don't have to maintain multiple instances, we can link to it from here.

Thoughts?

Reply:
I based it off what the android app provides - I'm not sure if it's the full range or just a subset?

I talked about making some stuff like this into blocks, so it can be repeated without clicking a link or managing multiple instances. Is that a better way?

Reply:
That would be pretty cool as well, I don't know if we've done that anywhere else. If you want to try it out let's see how it goes!